gnucash/gschemas/pref_transformations.xml

```diff
@@ -2152,16 +2152,20 @@
 <obsolete old-path="org.gnucash.window.pages.account-tree.summary"
           old-key="end-period"/>
 
+</release>
+
+
+<release version="5000">
+
```

Comment:
Why do you separate this out ?

Reply:
I just did this to make it clearer for me when testing, just to group the changes I made and maybe when that section is removed.